article/app/views/package.scala

```diff
@@ -36,6 +38,10 @@ object MainCleaner {
 object BodyCleaner {
   def apply(article: Article, html: String, amp: Boolean)(implicit request: RequestHeader) = {
     implicit val edition = Edition(request)
+
+    val shouldShowAds = !article.content.shouldHideAdverts && article.metadata.section != "childrens-books-site"
```

Comment:
Got it @jbreckmckye 
I guess the section check should be centralised somewhere instead of having this check scattered in multiple places in scala/javascript code. However I feel this goes behind the scope of this PR and could be the objective of another pull request